tests/test_tags.py

```diff
@@ -0,0 +1,76 @@
+import re
+from typing import Dict
+
+import pytest
+from packaging.version import Version
+
+from commitizen.tags import make_tag_pattern, tag_from_version
+
+TAG_FORMATS: Dict[str, Dict[str, list]] = {
```

Comment:
As a failsafe, add a case for the minimal `$version` that should always work:
```suggestion
TAG_FORMATS: Dict[str, Dict[str, list]] = {
	"$version": {
		"tags_per_version": [
			("1.2.3", "1.2.3"),
            ("1.2.3a2", "1.2.3a2"),
            ("1.2.3b2", "1.2.3b2"),
            ("1.2.3+1.0.0", "1.2.3+1.0.0"),
		],
		"invalid_tags": ["v1.2.3", "unknown-tag", "1-2-3", "ver1.0.0"]
	},
```